ts-payment-service/src/main/java/com/trainticket/controller/PaymentController.java

```diff
@@ -24,24 +26,59 @@ public class PaymentController {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PaymentController.class);
 
-    @GetMapping(path = "/welcome")
+    @ApiOperation(
+            value = "Service welcome / health-check",
+            notes = "Returns plain text used by load-balancers or monitoring tools to verify the service is alive."
+    )
+    @ApiResponses(@ApiResponse(code = 200, message = "Plain text welcome message",
+            response = String.class))
+    @GetMapping(path = "/welcome", produces = "text/plain")
     public String home() {
         return "Welcome to [ Payment Service ] !";
     }
 
-    @PostMapping(path = "/payment")
-    public HttpEntity pay(@RequestBody Payment info, @RequestHeader HttpHeaders headers) {
+    @ApiOperation(
+            value = "Pay for an order",
+            notes = "Debits the user’s balance and persists the payment record."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Payment successfully processed",
+                    response = Response.class)
+    })
+    @PostMapping(path = "/payment", consumes = "application/json", produces = "application/json")
+    public HttpEntity pay(
+            @ApiParam(value = "Payment payload", required = true) @RequestBody Payment info,
```

Comment:
Do the request/response entities also need to be annotated as part of this task?

In this PR only the endpoints are getting annotated, but all the data will still be plain.
Response entity that you're using is annotated, but doesn't contain any info on the actual response.